contrib/cirrus/mac_env.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# This script is intended to be called by Cirrus-CI on a Mac M1 persistent workier.
```

Comment:
Typo, 'workier'.

Also, in each header for each of these scripts, would you consider adding something like "this is step X of 3" and/or "this runs before/after X", so someone looking in the overflowing `contrib/cirrus ` directory can more easily figure out the order in which these run?

Reply:
Got it.  Sure thing, will do.  Fixes staged.